internal/txprocessor/txprocessor.go

```diff
@@ -135,11 +136,41 @@ func (r TXProcessor) txToBlock(ctx context.Context, tx relay.Transaction) (*neut
 	return &block, nil
 }
 
+// prepareHeaders returns two Headers for height and height+1 packed into *codectypes.Any value
+// We need two blocks in Neutron to verify both delivery of tx and inclusion in block:
+// - We need to know block X (`header`) to verify inclusion of transaction for block X (inclusion proof)
+// - We need to know block X+1 (`nextHeader`) to verify response of transaction for block X
+// since LastResultsHash is root hash of all results from the txs from the previous block (delivery proof)
+//
+// Arguments:
+// `height` - remote chain block height X = transaction with such block height
```

Comment:
Comment about arguments is copy pasted from the removed `FetchTrustedHeadersForHeights` method and wrong for the `prepareHeaders` method